client/core/core.go

```diff
@@ -3746,15 +3741,11 @@ func handlePreimageRequest(c *Core, dc *dexConnection, msg *msgjson.Message) err
 	if err != nil {
 		return fmt.Errorf("preimage send error: %v", err)
 	}
-	corder, cancelOrder := tracker.coreOrder()
-	var details string
+	subject := "preimage sent"
 	if isCancel {
-		corder = cancelOrder
-		details = fmt.Sprintf("match cycle has begun for cancellation order for trade %s", tracker.token())
-	} else {
-		details = fmt.Sprintf("match cycle has begun for order %s", tracker.token())
+		subject = "cancel preimage sent"
 	}
-	c.notify(newOrderNote("Preimage sent", details, db.Poke, corder))
+	c.notify(newOrderNote(subject, "", db.Data, tracker.coreOrder()))
```

Comment:
I've dropped the preimage notification from a poke severity to data severity. There's a lot of poke notes coming around that time, and I wanted to lighten up a little. I don't think the pre-image submission needs to be visible anyways, though it's been useful for development.